### PR TITLE
DOC: document daily / master branch strategy

### PR DESCRIPTION
**Pull request by matthew-brett**
From comment by Ralf at
https://github.com/MacPython/scipy-wheels/issues/6

### CODE REVIEW

**Comment by rgommers** [comment]
LGTM, merged. Thanks Matthew.

**Comment by rgommers** [comment]
This needs to also go into master. I'm about to push the beta1 build commit, can push this one after (don't want to clog up CI).

File: README.rst [hunk 1]
```diff
@@ -11,6 +11,35 @@ https://travis-ci.org/MacPython/scipy-wheels
...
+
+There are two important branches:
+
+* ``master`` - for building releases;
+* ``daily`` - for daily builds.
+
+Travis-CI builds the ``daily`` branch - er - daily, via a `Travis-CI cron job
+<https://docs.travis-ci.com/user/cron-jobs/>`_ to check that we can build
+against current Scipy master.   When trying to fix builds against master, or
+developing new CI build machinery, *please use the ``develop`` branch*.
```
Comment (pv) [inline README.rst]: There's no branch named `develop`?

Comment (matthew-brett) [reply README.rst]: Aha - yes - thanks.  I'll also add the instructions about Appveyor.  I forgot we had enabled that.

Comment (matthew-brett) [reply README.rst]: https://github.com/MacPython/scipy-wheels/pull/8

**Pull request merged**